src/frontend/src/lib/i18n/en.json

```diff
@@ -654,5 +654,64 @@
 				"open_source": "<strong>Open source</strong><br>$oisy_short is an <a rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://github.com/dfinity/oisy-wallet\">open-source project</a> lead by a dedicated team within the <a rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://dfinity.org/\">DFINITY foundation</a> supported by the organization’s 200 world-leading cryptographers and engineers."
 			}
 		}
+	},
+	"signer": {
+		"sign_in": {
+			"text": {
+				"access_your_wallet": "Access your $oisy_name",
```

Comment:
i don't know if you saw, but there is a new `$oisy_short` placeholder that is basically only "OISY", if useful

Reply:
Yes yes but the UI was reviewed with "OISY Wallet" so, just using it. Or did you spotted one sole "OISY" that should be changed?

Reply:
nono, it was really a question of OISY or OISY Wallet, but you explained, thank you!